src/modules/atommodel/serializer/RadixSerializer.ts

```diff
@@ -165,7 +165,7 @@ export class RadixSerializer {
     }
 
     public static toDSON(data: any): Buffer {
-        const enc = new cbor.Encoder()
+        const enc = new cbor.Encoder({ highWaterMark: 90000 })
```

Comment:
Maybe we should get in a comment here explaining what the `highWaterMark` thing is